packages/lab/src/slider/Slider.tsx

```diff
@@ -15,6 +15,15 @@ import { calculatePercentage, clamp, toFloat } from "./internal/utils";
 
 export interface SliderProps
   extends Omit<HTMLAttributes<HTMLDivElement>, "onChange" | "defaultValue"> {
+  /**
```

Comment:
will push up a fix for this.. "When bottom labels for minimum and maximum values are set"...

nothing else in the API speaks to bottom labels? So we should document this as

"When minimum and maximum labels are defined, ensure they are confined within the boundary of the slider."